src/components/ProjectPreview/index.js

```diff
@@ -10,33 +19,41 @@ const ProjectPreview = ({ project }) => {
 
   return (
     <Container>
-      <Overview>
-        <h1>{name.text}</h1>
-        <p>{description.text}</p>
-        <Link to={`/projects/${uid}`}>Watch</Link>
-      </Overview>
-      <Content>
-        {works.map((work) => {
-          const {
-            content: {
-              document: {
-                uid,
-                data: {
-                  title,
-                  embed: { thumbnail_url },
+      <FlexContainer>
+        <Overview>
+          <H1>{name.text}</H1>
+          <P>{description.text}</P>
+          <Link to={`/projects/${uid}`}>
+            <PrimaryButton color={YELLOW}>
+              <ButtonText bold>Watch</ButtonText>
+            </PrimaryButton>
+          </Link>
+        </Overview>
+        <Content>
+          {works.map((work) => {
+            const {
+              content: {
+                document: {
+                  uid,
+                  data: {
+                    title,
+                    embed: { thumbnail_url },
+                  },
                 },
               },
-            },
-          } = work;
-          return (
-            <div>
-              <img src={thumbnail_url} />
-              <h2>{title.text}</h2>
-              <Link to={`/works/${uid}`}>Watch</Link>
-            </div>
-          );
-        })}
-      </Content>
+            } = work;
+            return (
+              <div>
+                <img src={thumbnail_url} />
```

Comment:
need alt text here! you could leverage the title field and set the alt text using that